samples/javascript_es6/01.browser-echo/package.json

```diff
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "@babel/runtime": "^7.1.5",
-        "botbuilder-core": "~4.7.0",
+        "botbuilder-core": "~4.10.3",
```

Comment:
do we know this works?
the reason we were back at 4.7.0 was because of a version conflict with webchat - that might now be fixed @stevengum might be able to comment.

Reply:
Worked for me on Chrome and Edge, but I am not familiar with previous issues.

Reply:
That's good news! From what I remember the failure was obvious and fast, but let's wait for @stevengum  to sign off.

Reply:
I believe this was fixed via https://github.com/microsoft/BotBuilder-Samples/pull/1657